src/test/lambda/wizards/stateController.test.ts

```diff
@@ -188,16 +188,21 @@ describe('StateMachineController', function () {
         })
 
         it('handles branches', async function () {
-            const controller = new StateMachineController<{ branch1: string, branch2: string }>()
+            const controller = new StateMachineController<{ branch1: string; branch2: string }>()
```

Comment:
is your editor doing this? make sure you don't have an over-active vscode plugin ...

Reply:
I think this is prettier doing it? It probably didn't run on my PR since I cherry-picked the commit.